internal/api/apiservice/rebuild.go

```diff
@@ -120,18 +120,20 @@ func populateArtifact(ctx context.Context, t *rebuild.Target, mux rebuild.Regist
 }
 
 type RebuildPackageDeps struct {
-	HTTPClient                 httpx.BasicClient
-	FirestoreClient            *firestore.Client
-	Signer                     *dsse.EnvelopeSigner
-	GCBClient                  gcb.Client
-	BuildProject               string
-	BuildServiceAccount        string
-	UtilPrebuildBucket         string
-	BuildLogsBucket            string
-	BuildDefRepo               rebuild.Location
-	AttestationStore           rebuild.AssetStore
+	HTTPClient          httpx.BasicClient
+	FirestoreClient     *firestore.Client
+	Signer              *dsse.EnvelopeSigner
+	GCBClient           gcb.Client
+	BuildProject        string
+	BuildServiceAccount string
+	UtilPrebuildBucket  string
+	BuildLogsBucket     string
+	BuildDefRepo        rebuild.Location
+	AttestationStore    rebuild.AssetStore
+	// TODO: The local store and the debug store can be combined into a layered (cached) asset store.
```

Comment:
nit: want to shorten this TODO and add it at the end of the next line so this PR doesn't impact formatting so much?

Reply:
Moved to the top, couldn't really shorten it much but it doesn't affect formatting anymore.